{{cookiecutter.project_name}}/config.yaml

```diff
@@ -22,6 +22,8 @@ generator_args:
     mergeimports: true
   jsonldcontext:
     mergeimports: true
+  pydantic:
+    mergeimports: true
```

Comment:
@dalito - can you confirm that this addition does not result in the pydantic version of the python serialization being generated automatically for new projects and overwriting the already generated dataclasses serialization?  We definitely want to move to pydantic being the default python serialization eventually.

Reply:
Would it be OK to generate both standard python and pydantic? I have to try it out to be sure.

Reply:
a couple of repos I'm aware of do that, yep (renaming the pydantic file to something different from the dataclasses version e.g.:  https://github.com/biolink/biolink-model/blob/fe44b8d56b7a1e77fb1e94f78a143dddb8681565/Makefile#L127 -- the naming is kind of awkward but I bet we could come up with something more reasonable here). 

but, it might be that we just want to split this change out from this PR?  I will leave that to your judgment.

Reply:
I removed this change.